### PR TITLE
Add portlet of wallet settings

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: wallet-webapps-common/src/main/webapp/WEB-INF/conf/wallet/wallet-portal-configuration.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                            <string>wallet-common/WalletSettings</string>
+                        </field>
+                        <field name="permissions">
+                            <collection type="java.util.ArrayList">
+                                <value>
+                                    <string>*:/platform/users</string>
+                                </value>
+                                <value>
+                                    <string>*:/platform/externals</string>
+                                </value>
```
Comment (ahamdi) [inline wallet-webapps-common/src/main/webapp/WEB-INF/conf/wallet/wallet-portal-configuration.xml:33]: Please check if external users are allowed to have this setting with @mzorai

**Review by ahamdi — APPROVED** [review]

File: wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettingsDetails.vue [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      type: Object,
+      default: function() {
+        return null;
+      },
+    },
+  },
+  data: () => ({
+    id: `Wallet${parseInt(Math.random() * 10000)
+      .toString()
+      .toString()}`,
```
Comment (Rdidmohamedelhadi) [inline wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettingsDetails.vue]:
```suggestion
    id: `WalletSettingsDetails${parseInt(Math.random() * 10000)}`,
```

Comment (Rdidmohamedelhadi) [reply wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettingsDetails.vue]: @GouadriaHanen  please change this !

**Review by GouadriaHanen — APPROVED** [review]
LGTM

**Pull request merged**